config/systems.json

```diff
@@ -170,7 +170,6 @@
       },
       "nodejs": {
         "base_images": {
-          "10": "gcr.io/google-appengine/nodejs",
           "12": "gcr.io/google-appengine/nodejs",
```

Comment:
What's the situation on Azure/GCP?

Reply:
I checked the python, nodejs version supports on gcp, azure, openwhisk  official documentation most of them supports python versions 3.8 and above and for nodejs 18 and above.